activities.json

```diff
@@ -946,6 +946,18 @@
     "title": "Mixed Content",
     "url": "https://w3c.github.io/webappsec-mixed-content/"
   },
+  {
+    "ciuName": null,
+    "description": "The navigation API provides a web application-focused way of managing same-origin same-frame history entries and navigations.",
+    "id": "navigation-api",
+    "mozBugUrl": "https://bugzilla.mozilla.org/show_bug.cgi?id=1777171",
+    "mozPosition": "positive",
+    "mozPositionDetail": "There are various details that we're not sure about in the spec and we'd like to continue reviewing and submit feedback about issues we find. But the API as a whole is a good improvement for implementing SPAs over the status quo.",
```

Comment:
This opens with a lot of negativity for a positive position, maybe flip the ordering.